parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -292,9 +285,9 @@ public String toString() {
         + "Max row count for page size check is: " + getMaxRowCountForPageSizeCheck() + '\n'
         + "Truncate length for column indexes is: " + getColumnIndexTruncateLength() + '\n'
         + "Truncate length for statistics min/max  is: " + getStatisticsTruncateLength() + '\n'
-        + "Bloom filter enabled column names are: " + getBloomFilterColumns() + '\n'
+        + "Bloom filter enabled: " + bloomFilterEnabled + '\n'
         + "Max Bloom filter size for a column is " + getMaxBloomFilterBytes() + '\n'
-        + "Bloom filter enabled column expected number of distinct values are: " + getBloomFilterColumnExpectedNDVs().values() + '\n'
+        + "Bloom filter expected number of distinct values are: " + bloomFilterNDVs + '\n'
```

Comment:
are -> is

Reply:
It depends. `bloomFilterNDVs` is a map like structure that contains an NDV for each columns. The output of this line can be either one default value or several column name - NDV pairs.

Reply:
Sounds good. I thought earlier it was just one single number to allow the maximum NDVs, but I was wrong.